spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+require './lib/vehicle'
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+
+
+RSpec.describe VehicleFactory do
+  describe '#initialize' do
+    it 'instance of factory' do
+      factory = VehicleFactory.new
+
+      expect(factory).to be_an_instance_of(VehicleFactory)
+    end
+  end
+
+  describe '#ev_registrations' do
+    it 'has data' do
+      factory = VehicleFactory.new
+      wa_ev_registrations = DmvDataService.new.wa_ev_registrations
+      cars = factory.create_vehicles(wa_ev_registrations) 
```

Comment:
Again you can create another variable here or just write your expect statements like such
```ruby
expect(factory.create_vehicles(wa_ev_registrations)).to be_an(Array)
```